Kubernetes/windows/start-kubelet.ps1

```diff
@@ -211,35 +223,36 @@ else
     $hnsnetwork = Get-HnsNetwork | ? Type -EQ "L2Bridge"
 }
 
-    $podEndpointGW = Get-PodEndpointGateway $podCIDR
-    $hnsEndpoint = New-HnsEndpoint -NetworkId $hnsNetwork.Id -Name $endpointName -IPAddress $podEndpointGW -Gateway "0.0.0.0" -Verbose
-    Attach-HnsHostEndpoint -EndpointID $hnsEndpoint.Id -CompartmentID 1
-
-    netsh int ipv4 set int "$vnicName" for=en
-    #netsh int ipv4 set add "vEthernet (cbr0)" static $podGW 255.255.255.0
-    Start-Sleep 10
-    # Add route to all other POD networks
-    Update-CNIConfig $podCIDR
+$podEndpointGW = Get-PodEndpointGateway $podCIDR
+$hnsEndpoint = New-HnsEndpoint -NetworkId $hnsNetwork.Id -Name $endpointName -IPAddress $podEndpointGW -Gateway "0.0.0.0" -Verbose
+Attach-HnsHostEndpoint -EndpointID $hnsEndpoint.Id -CompartmentID 1
+
+netsh int ipv4 set int "$vnicName" for=en
+#netsh int ipv4 set add "vEthernet (cbr0)" static $podGW 255.255.255.0
+Start-Sleep 10
+# Add route to all other POD networks
+Update-CNIConfig $podCIDR
+
+$runtimeKubeletArgs = $commonKubeletArgs + @(
+    "--v=6",
+    "--allow-privileged=true",
```

Comment:
This flag has been removed in 1.15
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.15.md#action-required-1